x-pack/plugins/apm/server/routes/traces/get_trace_items.ts

```diff
@@ -9,26 +9,64 @@ import {
   QueryDslQueryContainer,
   Sort,
 } from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
-import { rangeQuery } from '@kbn/observability-plugin/server';
 import { ProcessorEvent } from '@kbn/observability-plugin/common';
+import { rangeQuery } from '@kbn/observability-plugin/server';
+import { APMConfig } from '../..';
 import {
+  AGENT_NAME,
+  CHILD_ID,
+  ERROR_EXCEPTION,
+  ERROR_GROUP_ID,
+  ERROR_ID,
   ERROR_LOG_LEVEL,
+  ERROR_LOG_MESSAGE,
+  EVENT_OUTCOME,
+  FAAS_COLDSTART,
   PARENT_ID,
+  PROCESSOR_EVENT,
+  SERVICE_ENVIRONMENT,
+  SERVICE_NAME,
+  SPAN_ACTION,
+  SPAN_COMPOSITE_COMPRESSION_STRATEGY,
+  SPAN_COMPOSITE_COUNT,
+  SPAN_COMPOSITE_SUM,
   SPAN_DURATION,
+  SPAN_ID,
+  SPAN_LINKS,
+  SPAN_NAME,
+  SPAN_SUBTYPE,
+  SPAN_SYNC,
+  SPAN_TYPE,
+  TIMESTAMP,
   TRACE_ID,
   TRANSACTION_DURATION,
+  TRANSACTION_ID,
+  TRANSACTION_NAME,
+  TRANSACTION_RESULT,
+  TRANSACTION_TYPE,
 } from '../../../common/es_fields/apm';
-import { getLinkedChildrenCountBySpanId } from '../span_links/get_linked_children';
+import {
+  WaterfallError,
+  WaterfallSpan,
+  WaterfallTransaction,
+} from '../../../common/waterfall/typings';
 import { APMEventClient } from '../../lib/helpers/create_es_client/create_apm_event_client';
-import { APMConfig } from '../..';
+import { getLinkedChildrenCountBySpanId } from '../span_links/get_linked_children';
+
+export interface TraceItems {
+  exceedsMax: boolean;
+  traceDocs: Array<WaterfallTransaction | WaterfallSpan>;
+  errorDocs: WaterfallError[];
+  linkedChildrenOfSpanCountBySpanId: Record<string, number>;
```

Comment:
German inspired naming? (eg. `Arbeiterunfallverischerungsgesetz`)

Reply:
I changed it to: `spanLinksCountById`